### PR TITLE
Inline icon shortcode and icon svg files. This PR contains a non-exha…

### PR DESCRIPTION
**Pull request by micronAMZN**
…ustive selection of common interface icon svg files, a shortcode for adding inline icons to documentation, and examples added to smoketest.md. These icons can be inlined in tutorials when instructing the user to choose a specific button. You can view the included icons by setting the smoketest.md draft to false in your local build.

Signed-off-by: Mike Cronin <mikecro@amazon.com>

### CODE REVIEW

**Comment by willihay** [comment]
FYI, another way to see smoketest.md is to include all draft docs in your local build:
`hugo server -D`

File: content/smoketest.md [hunk 1]
```diff
@@ -341,5 +341,73 @@ Indicates a section needs work, followed by a description of the task and a prom
...
+| prefab-edit.svg | {{< icon "prefab-edit.svg" >}} |
+| resolution.svg | {{< icon "resolution.svg" >}} |
+| rotate.svg | {{< icon "rotate.svg" >}} |
+| sc-bottom-align.svg | {{< icon "sc-bottom-align.svg" >}} |
+| sc-comment.svg | {{< icon "sc-comment.svg" >}} |
+| sc-group.svg | {{< icon "sc-group.svg" >}} |
+| sc-left-align.svg | {{< icon "sc-left-align.svg" >}} |
+| sc-right-align.svg | {{< icon "sc-right-align.svg" >}} |
+| sc-top-align.svg | {{< icon "sc-top-align.svg" >}} |
+| sc-ungroup.svg | {{< icon "sc-ungroup.svg" >}} |
```
Comment (willihay) [inline content/smoketest.md:394]: Using 'sc' in place of 'script-canvas' might not be intuitive for everyone, especially ESL contributors. I suggest you use 'script-canvas' as the prefix for these files.

**Review by willihay — CHANGES_REQUESTED** [review]

File: content/smoketest.md (hunk 1 above)
Comment (micronAMZN) [reply content/smoketest.md]: I have removed the sc prefix and renamed the files. Turns out these icons are general graph canvas icons and are used in other editors, not just Script Canvas.

**Review by willihay — APPROVED** [review]

**Review by chanmosq — APPROVED** [review]
Cool that we can use icons in our docs!

**Pull request merged**